stressberry/cli.py

```diff
@@ -129,6 +158,8 @@ def run(argv=None):
             "time": times,
             "temperature": temps,
             "cpu frequency": freqs,
+            "ambient": ambient,
+            "delta_t": delta_t,
```

Comment:
Let's not store `delta_t`. It can be computed easily enough from temp and ambient.

Reply:
Stored `delta_t `as it simplified plotting.  If we don't store it, then we will need to do list subtraction to calculate it before we can plot it. Believe this can be done with `numpy`, will need to investigate and test.

Reply:
ok got the plot working with `numpy` to calculate the ambient temperature. 
Can you clarify the intent of this code: 
```
    # sort the data such that the data series with the lowest terminal
    # temperature is plotted last (and appears in the legend last)
    terminal_temps = [d["temperature"][-1] for d in data]
    order = [i[0] for i in sorted(enumerate(terminal_temps), key=lambda x: x[1])]
```
If I understand this correctly, when multiple files are provided is looks at the final reading in each file and uses that to sort the legend. Is that correct?

Thinking about this since that would be after the idle period, with systems varying in how quickly they will heat up and cool down, I'm not sure that is the best/expected approach. 

Would it make more sense if the legend was ordered by which files had the highest temperature, which would typically occur during the stress period, not at the end of the test? Maybe I'm misunderstanding something?

Reply:
@nschloe Can you = explain this to me please?

Reply:
Guess we can cover this elsewhere.